internal/pkg/schema/generate.go

```diff
@@ -0,0 +1,182 @@
+package schema
+
+import (
+	"math"
+	"sort"
+
+	"github.com/iancoleman/orderedmap"
+	"github.com/santhosh-tekuri/jsonschema/v5"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func GenerateDocument(schemaDef []byte) (string, error) {
```

Comment:
Tato metoda vygeneruje na zaklade JSON schemy prazdny JSON dokument, ktory obsahuje kluce zo schemy a defaultne hodnoty.

Bude sa to pouzvat vramci prikazu, ktory vytvara novy config/row.
Aby uzivatel nedostal uplne prazdny JSON, ale mal tam predvyplnene kluce (ak ma komponenta zadanu JSON schemu).